README.md

```diff
@@ -1,5 +1,9 @@
 # HEPData pyhf extractor
 
+![GH Actions status][badge-actions-status]
+[![Coverage status](https://coveralls.io/repos/github/HEPData/hepdata-validator/badge.svg)][badge-coverage-status]
```

Comment:
Typo: `hepdata-validator` → `hepdata-pyhf-extractor` and add `?branch=main` at end of URL.

Reply:
Done in 4d6e800a8495c954276105a8c3dc8f192c77f4d8

Reply:
Good catch!